pkg/operator/apiserver/controllerset/apiservercontrollerset.go

```diff
@@ -313,6 +314,7 @@ func (cs *APIServerControllerSet) WithEncryptionControllers(
 	provider controllers.Provider,
 	deployer statemachine.Deployer,
 	migrator migrators.Migrator,
+	rawUnsupportedConfigOverrideFunc config.RawUnsupportedConfigFunc, /*optional can be nil*/
```

Comment:
what does it mean if it is nil?

Reply:
can we turn this into `WithCustomUnsupportedConfigFunc`?

Reply:
if it is nil we will get the config from `operator.spec.UnsupportedConfigOverrides.Raw`

Reply:
Wondering whether it isn't much simpler and enough to pass a json path?